### PR TITLE
RSARTE-175884: Finalize GCC 12 support

### PR DESCRIPTION
**Pull request by rajneshkumarmalik**
1. Removed using namespace std and directly used std with cout and string because it was conflicting byte data type with poco lib.
2. Modified prebuild_MSVS-17-x64.tcjs to support poco-1.12.4 lib.
3. Added tc.library field in tcpServerLib.tcjs.

### CODE REVIEW

File: LibTCPServer.emx [hunk 1]
```diff
@@ -366,13 +366,13 @@ srv->start();
 	}&#xD;
 	Object::Ptr object = result.extract&lt;Object::Ptr>();&#xD;
 	Var var_cmd = object->get(&quot;command&quot;);&#xD;
-	string cmd = var_cmd.convert&lt;string>();&#xD;
+	std::string cmd = var_cmd.convert&lt;std::string>();&#xD;
 	if (config.logToStdOut)&#xD;
-		cout &lt;&lt; &quot;JSON command received: &quot; &lt;&lt; cmd &lt;&lt; endl &lt;&lt; flush;&#xD;
```
Comment (elena-volkova-hcl) [inline LibTCPServer.emx:371]: Why did you remove flush. It also comes from std. please, keep it.

File: LibTCPServer.emx [hunk 2]
```diff
@@ -668,7 +668,7 @@ if( port != 0 &amp;&amp; port->getName() != 0) {&#xD;
 str.append(&quot;}&quot;);&#xD;
 &#xD;
 if (config.logToStdOut)&#xD;
-	cout &lt;&lt; &quot;Out message: &quot; &lt;&lt; str &lt;&lt; endl &lt;&lt; flush;&#xD;
```
Comment (elena-volkova-hcl) [inline LibTCPServer.emx:671]: Keep flush

File: LibTCPServer.emx [hunk 3]
```diff
@@ -1048,8 +1048,8 @@ catch (Poco::Exception&amp; exc) {&#xD;
...
+    <headerPreface xmi:id="_5rMywu5TEeitL-zKjjNp2A" body="#include &lt;string>&#xD;&#xA;//using namespace std;&#xD;&#xA;namespace Poco {&#xD;&#xA;&#x9;namespace Net {&#xD;&#xA;&#x9;&#x9;class TCPServer;&#xD;&#xA;&#x9;}&#xD;&#xA;}&#xD;&#xA;"/>
```
Comment (elena-volkova-hcl) [inline LibTCPServer.emx]: Do not comment the line, please remove it completely.

File: tcpServerLib.tcjs [hunk 4]
```diff
@@ -15,6 +15,7 @@ tc.inclusionPaths = topTC.inclusionPaths || [
     pocoLoc + '\\JSON\\include',
     pocoLoc + '\\Foundation\\include'
 ];
+tc.libraries = topTC.libraries;
```
Comment (elena-volkova-hcl) [inline tcpServerLib.tcjs]: I think this is not needed.

**Review by elena-volkova-hcl — CHANGES_REQUESTED** [review]
Please keep std::flush
Do not comment out using namespace std, but remove the line completely
Check tcjs change, I do not think it is needed.

File: LibTCPServer.emx (hunk 1 above)
Comment (rajneshkumarmalik) [reply LibTCPServer.emx:371]: The following link indicated that endl internally having new line followed by flush. I think adding one more flush is not required. Let me know you view on this.
https://en.cppreference.com/w/cpp/io/manip/endl

File: LibTCPServer.emx (hunk 3 above)
Comment (rajneshkumarmalik) [reply LibTCPServer.emx]: I will remove it. i missed it.

**Review by mattias-mohlin — APPROVED** [review]
Ok. I have no other comments except those Elena already gave, except that it would be easier to review the changes if you turn on the preference Modeling - Use CDATA when saving model files (to get rid of all XML encoded characters in code snippets).

File: tcpServerLib.tcjs (hunk 4 above)
Comment (rajneshkumarmalik) [reply tcpServerLib.tcjs]: ok, I will remove it.
